src/__experimental__/components/decimal/decimal.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, text, number, select
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import Decimal from './decimal.component';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import notes from './documentation';
+
+const store = new Store({
+  value: Decimal.defaultProps.value
+});
+
+const setValue = (evt) => {
+  action('onChange')(evt);
+  store.set({ value: evt.target.value });
+};
+
+storiesOf('Experimental/Decimal', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [State]
```

Comment:
you could also add the `Textbox` props from the `Textbox` component

Reply:
I don't think I need to? The `Textbox` props aren't being shown by the `Info` addon?

Reply:
yes, but they're specified in the `Decimal` component and then passed to the `Textbox` component, so I think they should be displayed for the documentation purpose.